client-course-schedulizer/src/utilities/services/facultyLoadsService.ts

```diff
@@ -57,7 +57,7 @@ const updateRow = ({
       : (prevRow[termCourseSectionProp] = sectionName);
 
     prevRow[termHoursProp] = prevRow[termHoursProp]
-      ? Number(prevRow[termHoursProp]) / section.instructors.length + facultyHours
+      ? Number(prevRow[termHoursProp]) + facultyHours
```

Comment:
This was causing a miscalculation of some faculty loads. This division already occurs in the calculation of `facultyHours` so it was unnecessary.